bedrock/urls_pocket_mode.py

```diff
@@ -0,0 +1,33 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+from django.conf import settings
+from django.urls import include, path
+from django.utils.module_loading import import_string
+
+from watchman import views as watchman_views
+
+from bedrock.base import views as base_views
+
+# The default django 404 and 500 handler doesn't run the ContextProcessors,
+# which breaks the base template page. So we replace them with views that do!
+handler500 = "bedrock.base.views.server_error_view"
+handler404 = "bedrock.base.views.page_not_found_view"
+
+
+urlpatterns = (
+    # Main pages
+    # path("", include("bedrock.sitemaps.urls")), ## TODO sort out a pocket sitemap
```

Comment:
At the moment Pocket appears to have no sitemap, and it's not included in the main mozorg sitemap (for good reason), so we should probably take the opportunity to add a simple one and serve that, too.

Reply:
I don't think we need a sitemap for pocket mode, as most pages on the domain will be not under our control, they'll just be proxied.